pandas/io/json/normalize.py

```diff
@@ -281,6 +281,6 @@ def _recursive_extract(data, path, seen_meta, level=0):
             raise ValueError('Conflicting metadata name {name}, '
                              'need distinguishing prefix '.format(name=k))
 
-        result[k] = np.array(v).repeat(lengths)
+        result[k] = np.array(v, dtype=object).repeat(lengths)
```

Comment:
Not saying this is right or wrong just surprised this is what solves the issue. What dtype was this producing before?

Reply:
It was returning a string